deployment/hasura/metadata/databases/AerieMerlin/tables/metadata/users_and_roles_view.yaml

```diff
@@ -0,0 +1,16 @@
+table:
+  name: users_and_roles
+  schema: metadata
+configuration:
+  custom_name: "users_and_roles"
+select_permissions:
+  - role: user
+    permission:
+      columns: [id, username, hasura_default_role, hasura_allowed_roles]
+      filter: {"id":{"_eq":"X-Hasura-User-Id"}}
+      allow_aggregations: false
```

Comment:
I'm observing that we sometimes set `allow_aggregations` to true and sometimes to false. Do we have a general feel for when it should be one or the other?

Reply:
In general, I allow aggregations to be `true`. I've only set it to false where a role has a filter on their `SELECT` privileges (or in the case of `users_and_roles_view` where they are only allowed to SELECT one row). Since aggregates only run on the portion of the table they're allowed to select, I can set the `users_allowed_roles` to allowing aggregates, although I stand by them remaining disable for `users_and_roles_view`.